Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04 as monero-lws-build
+
+# Set monerod and monero-lws versions to install from github
+ARG MONERO_BRANCH=v0.17.3.0
+ARG MONERO_COMMIT_HASH=ab18fea3500841fc312630d49ed6840b3aedb34d
+
+ARG MONERO_LWS_BRANCH=release-v0.1_0.17
+ARG MONERO_LWS_COMMIT_HASH=9d8a7e558aace21782217f69c3f083069538f38a
+
+# install dependencies
+RUN set -o errexit -o nounset \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get upgrade -y -q \
+    && apt-get install -y -q git build-essential cmake pkg-config libssl-dev libzmq3-dev libunbound-dev libsodium-dev libunwind8-dev \
+    liblzma-dev libreadline6-dev libldns-dev libexpat1-dev libpgm-dev qttools5-dev-tools libhidapi-dev libusb-1.0-0-dev \
```

Comment:
add `jq`

Reply:
`jq` is _not_ needed to run, but is useful for the bash commands listed in the documentation. I think it makes more sense to omit it as a default installed component, but I am also less familiar with docker norms.

Reply:
I was thinking a good next step for this PR would be basing the image on Alpine to reduce size and complexity, as well as moving to a multi-step build where the built binaries (admin and the daemon?) are essentially copy/pasted into a subsequent build process that only adds 'necessary' packages for running and using these binaries- this is where I would probably put jq... 

I'm not super familiar with LWS, but if there aren't any RPC methods for admin stuff and jq is the easiest current way to do these maintenance tasks (it appears to be the documented way), then I would probably support adding it in.  Otherwise, everyone that reads the current documentation will have to manually install it within the running container every time it restarts...  and they probably will do that. 

There's a P2Pool Dockerfile that basically does this exact process (including adding jq at the end): https://github.com/WeebDataHoarder/p2pool-compose/blob/master/docker/p2pool/Dockerfile in case you wanto to check it out.

Reply:
I'd definitely recommend adding jq in build if it's at all necessary to normal operations and admin functions for LWS (which it seems to be), as installing packages after build is usually quite difficult as the running user in the container should *not* be root or have sudo permissions.